src/scripts/main.js

```diff
@@ -2,7 +2,36 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(person => {
+  person.sex = sexData(person.sex);
```

Comment:
```suggestion
  person.sex = person.sex === 'm' ? 'Male' : 'Female';
```
You can just use ternary operator